src/modules/commands/common/membercount.ts

```diff
@@ -0,0 +1,19 @@
+import { Command } from '@commands/CommandHandler'
+import { censusApi } from '@planetside/CensusApi'
+import { constants } from '@app/global/constants'
+
+export default new Command({
```

Comment:
Command isn't included in commands/common/index.ts.

Reply:
ok